src/test/java/core/basesyntax/DailyReportWriterImplTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.DailyReportWriter;
+import core.basesyntax.service.impl.DailyReportWriterImpl;
+import java.io.File;
+import org.junit.Test;
+
+public class DailyReportWriterImplTest {
```

Comment:
add @After method, and clear storage after each test, because i can see that some tests dont clear after